src/locales/locale-en.json

```diff
@@ -116,6 +116,13 @@
     "TITLE": "Feedback",
     "TOGETHER_WITH": "You did this together with:"
   },
+
+  "ACTIVITY_FEEDBACK_DISMISS": {
```

Comment:
I think this would be better inside the `ACTIVITY_FEEDBACK` object, doesn't feel like a big enough concern to justify a new top level object...

So, for example, could be:
- `ACTIVITY_FEEDBACK.DISMISS_LABEL`
- `ACTIVITY_FEEDBACK.DISMISS_TITLE`
- `ACTIVITY_FEEDBACK.DISMISS_MESSAGE`

(also making them less coupled to the UI components, ~~button~~, ~~prompt~~)